File: TransCoder_tokenized_test_set_functions/cpp/GIVEN_TWO_UNSORTED_ARRAYS_FIND_PAIRS_WHOSE_SUM_X_1.cpp
void findPairs ( int arr1 [ ], int arr2 [ ], int n, int m, int x ) {
  unordered_set < int > s;
  for ( int i = 0;
  i < n;
  i ++ ) s . insert ( arr1 [ i ] );
  for ( int j = 0;
  j < m;
  j ++ ) if ( s . find ( x - arr2 [ j ] ) != s . end ( ) ) cout << x - arr2 [ j ] << " " << arr2 [ j ] << endl;
}

File: TransCoder_tokenized_test_set_functions/python/GIVEN_TWO_UNSORTED_ARRAYS_FIND_PAIRS_WHOSE_SUM_X_1.py
def findPairs ( arr1, arr2, n, m, x ) :
    s = set ( )
    for i in range ( 0, n ) :
        s . add ( arr1 [ i ] )
    for j in range ( 0, m ) :
        if ( ( x - arr2 [ j ] ) in s ) :
            print ( ( x - arr2 [ j ] ), '', arr2 [ j ] )
    
